tensorflow/compiler/mlir/tools/kernel_gen/transforms/tf_to_jit_invocations.cc

```diff
@@ -115,20 +119,33 @@ struct TFToI64JITInvocationForLargeTensorsPattern : public RewritePattern {
 
   LogicalResult matchAndRewrite(Operation *op,
                                 PatternRewriter &rewriter) const override {
-    if (!IsUnaryTFOperation(op) ||
+    if ((!IsUnaryTFOperation(op) && !IsBinaryTFOperation(op)) ||
```

Comment:
Maybe this could be one function, `IsNaryTFOperation(int)`

Reply:
I think we better keep them seperate as the dispatcher code requires the binary case to add to the operation based on the second case.